src/strand-datepicker-panel/example.html

```diff
@@ -0,0 +1,4 @@
+<strand-button id="btn">
+	<label>Select a Date</label>
+</strand-button>
+<strand-datepicker id="dp" target="#btn" start-enabled end-enabled use-timezone="false" use-range="false"></strand-datepicker>
```

Comment:
shouldn't this be panel as well? also i'm not 100% that this should be public since its really just an abstraction